docs: fix invalid links in markdown files (#756)
asyncapi/website: pages/blog/status-update-41-20.md

    byline: AsyncAPI Maintainer and Community Guardian
---

## New Website

> I'm super excited to share with you that last week we released [a new AsyncAPI website](/):

Reviewer: ```suggestion
I'm super excited to share with you that last week we released [a new AsyncAPI website](https://www.asyncapi.com/):
```

This was giving a 404

> - On a home page, we now promote [the Playground](https://playground.asyncapi.io) more and put much more emphasis on the different ways you can engage with AsyncAPI Community,

Reviewer: ```suggestion
- On a home page, we now promote [Studio](https://studio.asyncapi.com) more and put much more emphasis on the different ways you can engage with AsyncAPI Community,
```

- Documentation view got additional navigation per document so, for example, it is now much better to navigate through different sections of [the specification document](/docs/specifications/2.0.0),

> - The tools section is now dedicated to AsyncAPI's official tools to clarify the AsyncAPI initiative is not just the spec. Just have a look how much detailed is, for example, the [generator view](/tools/generator),

Reviewer: ```suggestion
- The tools section is now dedicated to AsyncAPI's official tools to clarify the AsyncAPI initiative is not just the spec. Just have a look how much detailed is, for example, the [generator view](https://www.asyncapi.com/tools/generator),
```

This is broken and 404s

> - Don't worry, all the other tools are still there, under [the Community section](/docs/community/tooling)

Reviewer: ```suggestion
- Don't worry, all the other tools are still there, under [the Tools section](https://github.com/asyncapi/website/tree/master/pages/tools)
```
this 404s

> - The cherry on the cake is a much more appealing [Blog view](/blog)

Reviewer: ```suggestion
- The cherry on the cake is a much more appealing [Blog view](https://www.asyncapi.com/blog)
```

another 404

    byline: AsyncAPI Maintainer and Community Guardian
---

## New Website

I'm super excited to share with you that last week we released [a new AsyncAPI website](https://www.asyncapi.com/):

- On a home page, we now promote [Studio](https://studio.asyncapi.com) more and put much more emphasis on the different ways you can engage with AsyncAPI Community,
- Documentation view got additional navigation per document so, for example, it is now much better to navigate through different sections of [the specification document](/docs/specifications/2.0.0),
- The tools section is now dedicated to AsyncAPI's official tools to clarify the AsyncAPI initiative is not just the spec. Just have a look how much detailed is, for example, the [generator view](https://www.asyncapi.com/tools/generator),

> - Don't worry, all the other tools are still there, under [the Tools section](https://github.com/asyncapi/website/tree/master/pages/tools)

Reviewer 2: ```suggestion
- Don't worry, all the other tools are still there, under [the Tools section](https://github.com/asyncapi/website/tree/master/pages/tools)
```

I prefer to have this link - https://www.asyncapi.com/docs/tools to show list of tools to user.

Do we regret we engaged with event participants? Hell no! So far, we got only 2 spam PRs, but we also got many issues addressed (26 in 7 days) that we listed for the event.

We had two primary goals to join the event:

- Show the community that AsyncAPI Initiative is not just the spec but a lot of great tooling.
- Help out people to take their first baby steps in open-source contributions with a welcoming community. We not only prepared issues with different levels of difficulty but also created a set of videos that explain how to contribute and what the issues are about.

Anyway. We share the summary of how Hacktoberfest ended up for us in the next status update. So far, so good though.

Remember that throughout the entire October me and Fran, we'll be hosting office hours so anyone can join and ask for help or even do a pair programming session with us. Feel invited! More details in the blog post about the event. We will stream to our official media accounts:

- https://www.twitch.tv/asyncapi
- https://www.youtube.com/asyncapi
- https://twitter.com/AsyncAPISpec

> Look into [this](https://calendar.google.com/calendar/u/0?cid=dGJyYmZxNGRlNWJjbmd0OG9rdmV2NGxzdGtAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ) calendar for the schedule.

Reviewer 2: Update the calendar link for the schedule of the meetings.
